src/llamacrawl/services/readers/elasticsearch.py

```
@@ -85,21 +86,19 @@ def __init__(
             self.es_version = info["version"]["number"]
             self.logger.info(
                 f"Connected to Elasticsearch {self.es_version}",
-                extra={"source": self.source_name, "es_version": self.es_version}
+                extra={"source": self.source_name, "es_version": self.es_version},
             )
         except Exception as e:
             self.logger.error(
                 f"Failed to connect to Elasticsearch at {es_url}: {e}",
-                extra={"source": self.source_name, "error": str(e)}
+                extra={"source": self.source_name, "error": str(e)},
             )
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)